source/gloperate-text/source/stages/GlyphRenderStage.cpp

```diff
@@ -46,9 +46,11 @@ void GlyphRenderStage::onProcess(gloperate::AbstractGLContext *)
 
     auto fbo = targetFramebuffer.value();
 
+    std::unique_ptr<globjects::Framebuffer> resource_wrapper;
     if (!fbo)
     {
-        fbo = globjects::Framebuffer::defaultFBO();
+        resource_wrapper = globjects::Framebuffer::defaultFBO();

```

Comment:
The ownership of the default FBO is a problem in almost all stages.
Solutions are:

1.  resource managing object with function scope (GlyphRenderStage)
2. a member in every stage that needs default FBO (MixerStage, BlitStage)
3. a member in gloperate::stage
4. make default FBO a singleton